cmd/kubeadm/app/preflight/checks.go

```diff
@@ -878,6 +878,8 @@ func RunInitMasterChecks(execer utilsexec.Interface, cfg *kubeadmapi.InitConfigu
 		PortOpenCheck{port: int(cfg.LocalAPIEndpoint.BindPort)},
 		PortOpenCheck{port: 10251},
```

Comment:
@joshrosso - any update?

Reply:
@timothysc not yet. I think this PR is blocked on the changes luxas suggested in an above comment.

I know what needs to be done, I just need to find the time to do them. I won’t have any availability for about 2 weeks. But would like to tackle this soon if no one gets it before me.